README.md

```diff
@@ -251,7 +251,22 @@ Example run customization:
                 "labels": {
                     "label1": "value1",
                     "label2": "value2"
-                }
+                },
+                "ports": [
```

Comment:
The new properties should be documented in the table above.  They should also be added to the [JSON Schema for the configuration](https://github.com/Microsoft/vscode-docker/blob/34b406052e3224817421317e2de34b7e275b4145/package.json#L383) in the `package.json`.